firewall/handlers/handlers_util.go

```diff
@@ -0,0 +1,15 @@
+package handlers
+
+func removeDuplicates(xs *[]string) {
```

Comment:
too hard to read this function. Here `xs`, `i`, `x`, `j` four short unreadable variables used in 4 rows. 
would be great to add tests for this function

Reply:
it is looking for exact string match, right? (for now)

Reply:
Agree. Test is added

Reply:
@vixentael yes